src/extensions/default/QuickView/main.js

```diff
@@ -784,10 +812,15 @@ define(function (require, exports, module) {
 
     // Setup initial UI state
     setEnabled(prefs.get("enabled"), true);
+    setExtensionlessImagePreview(prefs.get("extensionlessImagePreview"));
```

Comment:
Actually, call to `setEnabled(prefs.get("enabled"), true);` on line above is not writing to prefs, so that should be ok.